### ISSUE TITLE
Cannot read properties of null (reading 'default')

### ISSUE THREAD

**Issue by leiqing110** [microsoft/nni#4847]
**Describe the issue**:

![image](https://user-images.githubusercontent.com/19700402/167283952-d6abdb89-8861-489f-99f3-82b7d3fdc58a.png)

**Environment**:
- NNI version:
- Training service (local|remote|pai|aml|etc):
- Client OS:
- Server OS (for remote mode only):
- Python version:
- PyTorch/TensorFlow version:
- Is conda/virtualenv/venv used?:
- Is running in Docker?:

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log:
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**How to reproduce it?**:

**Comment by Lijiaoa** [comment]
Thanks for your detailed message for debug! maybe the wrong final result cause this problem. Could you paste your final result?
`[2022-05-07 13:19:12] ERROR (nni.runtime.msg_dispatcher_base/Thread-1) Incorrect final result: the final result should be float/int, or a dict which has a key named "default" whose value is float/int.` https://nni.readthedocs.io/en/stable/experiment/web_portal/web_portal.html#what-should-you-do-when-you-think-the-chart-is-strange-such-as-default-metric-hyper-parameter

**Comment by leiqing110** [comment]
> 

I don't have a problem using one GPU per trail, but when a trail uses 2 GPUs, this problem arises, is there a problem with my congfig settings?

**Comment by Lijiaoa** [comment]
hi @leiqing110  Do you still have these problem with v2.9?
